tests/single/single_test.js

```diff
@@ -1,12 +1,15 @@
 module.exports = {
-  'Google\'s Search Functionality' : function (browser) {
+  "Google's Search Functionality": function (browser) {
     browser
-      .url('https://www.google.com/')
-      .waitForElementVisible('body', 1000)
-      .setValue('input[type=text]', 'BrowserStack')
-      .click('input[name=btnK]')
+      .url("https://www.google.com")
+      .setValue("input[name=q]", ["BrowserStack", browser.Keys.ENTER]) // this submits on desktop browsers
       .pause(1000)
-      .assert.title('BrowserStack - Google Search')
-      .end();
-  }
+      .title((result) => {
+        if (!/BrowserStack/i.test(result.value)) {
+          browser.submitForm("input[name=q]"); // this helps in mobile browsers
+        }
+      })
+      .expect.title().to.match(/BrowserStack/i);
```

Comment:
```suggestion
      }).expect.title().to.match(/BrowserStack/i);
```
chained commands with an already ending block in the previous line can be written the following way.

Reply:
similarly at other places if any